### PR TITLE
Fix: Build enable trace facility

### PR DESCRIPTION
**Pull request by alfred2g**
Fix: Build enable trace facility

Description
-----------
Fix: Build enable trace facility

Test Steps
-----------
github actions pass

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by Skptak — APPROVED** [review]

**Review by paulbartell — APPROVED** [review]

File: FreeRTOS/Demo/CORTEX_MPS2_QEMU_IAR_GCC/FreeRTOSConfig.h [hunk 1]
```diff
@@ -39,6 +39,12 @@
  * See http://www.freertos.org/a00110.html
  *----------------------------------------------------------*/
 
+#define configUSE_TRACE_FACILITY 1
+#define configGENERATE_RUN_TIME_STATS 1
+
+#define portCONFIGURE_TIMER_FOR_RUN_TIME_STATS()
+#define portGET_RUN_TIME_COUNTER_VALUE() 0
```
Comment (jasonpcarroll) [inline FreeRTOS/Demo/CORTEX_MPS2_QEMU_IAR_GCC/FreeRTOSConfig.h:46]: Setting these to blank will make run time stats meaningless - why not just remove any calls that need these macros and disable generating run time stats. I'm not overly familiar with the capabilities of QEMU but if QEMU can emulate a timer, that should be done here.

**Review by jasonpcarroll — APPROVED** [review]

File: FreeRTOS/Demo/CORTEX_MPS2_QEMU_IAR_GCC/FreeRTOSConfig.h [hunk 2]
```diff
@@ -39,6 +39,12 @@
  * See http://www.freertos.org/a00110.html
  *----------------------------------------------------------*/
 
+#define configUSE_TRACE_FACILITY 1
```
[the remark this answers is not in the document]
Comment (alfred2g) [reply FreeRTOS/Demo/CORTEX_MPS2_QEMU_IAR_GCC/FreeRTOSConfig.h:42]: ah no, this is not related to trace, it is just enabling a few functions in the kernel which provide some debugging functionalities such as printing the task list

File: FreeRTOS/Demo/CORTEX_MPS2_QEMU_IAR_GCC/FreeRTOSConfig.h (hunk 1 above)
Comment (alfred2g) [reply FreeRTOS/Demo/CORTEX_MPS2_QEMU_IAR_GCC/FreeRTOSConfig.h]: Good point, pushed a fix for that

**Comment by aggarg** [comment]
Please update the PR description with details about what was failing and what is fixed in this PR.

**Review by kstribrnAmzn — APPROVED** [review]

**Review by cobusve — APPROVED** [review]

**Pull request merged**